Sources/StreamChat/Controllers/ChannelController/ChannelController+Capabilities+Features.swift

```diff
@@ -0,0 +1,188 @@
+//
+// Copyright © 2023 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+// MARK: - Channel current user capabilities
+
+public extension ChatChannelController {
```

Comment:
Maybe these should be moved on the channel level? I see a lot of usages directly from the channel, where we repeat the same code.

Reply:
Agree with that

Reply:
We should be deprecating the ones that were previously exposed in ChannelController IMO

Reply:
I thought about that too, but the reason for this decision was related to the following:
1. To be consistent with the other ones we already had (ex: channelController?.areRepliesEnabled).
2. It makes more sense to live in the `channelController` because it is here that we can perform actions, so `channel.canReply` could not be 100% clear what it means 🤔 
3. This was just a syntax sugar to make it easier to use it from the channelController, because from the channel, it is not that bad, and `channel.ownCapabilities.contains(.sendReply)` is kinda fine.

For me, I'm fine with both ways. Changing this will require some changes in tests etc. Let me know if your decision remains the same and you guys want me to change this 👍

Reply:
> We should be deprecating the ones that were previously exposed in ChannelController IMO

@polqf You mean move those ones to the `ChatChannel`?

Reply:
we could move them to the `ChatChannel` and call those from the `ChannelController` and from other places in the code.

Reply:
+1 on what Martin said